www/form/inner.js

```diff
@@ -4260,12 +4258,49 @@ define([
                     }
                 }
             }
+            let shiftButtons;
+            let drag;
+            if (editable) {
+                drag = $('#toggle-drag-off').attr('class').indexOf('toggle-active') !== -1 ? false : true;
```

Comment:
If you merge your `if` block with the existing one, the `drag` state is already defined above